README.md

```diff
@@ -120,6 +120,10 @@ The following events are slated for removal from contrib-ads and will have no sp
 * `adscanceled` (EVENT) -- In the future, this event will no longer result in special behavior in contrib-ads. It was intended to cancel all ads, but it was never fully implemented. Instead, trigger `nopreroll` and `nopostroll`.
 * `adserror` (EVENT) -- In the future, this event will no longer result in special behavior in contrib-ads. Today, this event skips prerolls when seen before a preroll ad break. It skips postrolls if seen after contentended and before a postroll ad break. It ends linear ad mode if seen during an ad break. These behaviors should be replaced using `skipLinearAdMode` and `endLinearAdMode` in the ad integration.
 
+## Autoplay
+
+Using the `autoplay` attribute on the video element is not supported. Due to Autoplay Restrictions [introduced](https://developers.google.com/web/updates/2017/09/autoplay-policy-changes) by [browsers](https://webkit.org/blog/7734/auto-play-policy-changes-for-macos/), using the `autoplay` attribute on the player will often not behave as expected. The recommended [best practice](https://developers.google.com/web/updates/2017/09/autoplay-policy-changes#best-practices) is to use the `player.play()` method to autoplay when autoplay without user interaction is not restricted and have access to the [play promise](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/play) if it is available on that browser.
```

Comment:
For the webkit.org link maybe say "by browsers such as [Safari]" so it's clear the link is about Safari.